skimage/restoration/__init__.pyi

```diff
@@ -0,0 +1,37 @@
+# Explicitly setting `__all__` is necessary for type inference engines
+# to know which symbols are exported. See
+# https://peps.python.org/pep-0484/#stub-files
+
+__all__ = ['wiener',
+           'unsupervised_wiener',
+           'richardson_lucy',
+           'unwrap_phase',
+           'denoise_tv_bregman',
+           'denoise_tv_chambolle',
+           'denoise_bilateral',
+           'denoise_wavelet',
+           'denoise_nl_means',
+           'denoise_invariant',
+           'estimate_sigma',
+           'inpaint_biharmonic',
+           'cycle_spin',
+           'calibrate_denoiser',
+           'rolling_ball',
```

Comment:
Looks good,thanks. I think [on MacOS this function is again confused with the submodule of the same name](https://github.com/scikit-image/scikit-image/actions/runs/5253063227/jobs/9489909042#step:6:536). :roll_eyes: Could you prefix `skimage/restoration/rolling_ball.py` accordingly with a `_` like in previous PRs?

Reply:
And feel welcome to do the same for the other "public" submodules in `skimage/restoration`. They should be private.

Reply:
Sure, I will update and refactor changes.